google/cloud/storage/examples/storage_default_object_acl_samples.cc

```diff
@@ -73,14 +80,21 @@ void CreateDefaultObjectAcl(google::cloud::storage::Client client, int& argc,
   auto role = ConsumeArg(argc, argv);
   //! [create default object acl] [START storage_add_default_owner]
   namespace gcs = google::cloud::storage;
+  using google::cloud::StatusOr;
   [](gcs::Client client, std::string bucket_name, std::string entity,
      std::string role) {
-    gcs::ObjectAccessControl result =
+    StatusOr<gcs::ObjectAccessControl> result =
```

Comment:
For naming convention, could you use `default_object_acl` instead of `result`. It helps with context of the variable.